docs/module_docs/featdet_resolver_local_minimum/local-minimum-resolver.md

```diff
@@ -1,109 +1,88 @@
 
-Mobility has to be selected as a dimension to resolve mobilograms [1].
+2. [_Mobilograms_](../../terminology/ion-mobility-terminology.md#mobilograms) are recalculated from  raw data, even though a [smoothing](../featdet_smoothing/smoothing.md) step was previously applied. Non-smoothed _mobiligrams_ tend to be more jagged than regular EIC traces (see Figure). Threfore, some parameters (_e.g._ [Min search range](#minimum-search-range-rtmobility-absolute) and (_e.g._ [Min ratio of peak top/edge](#min-ratio-of-peak-topedge)) should be adjusted accordingly.
 
-##### Chromatographic threshold
+![Moobility resolving](mobility-resolving.png)
 
-Since there are less scans in mobility dimension (e.g., 400 - >1000 per frame, depending on the
-instrument type and setting) than in rt dimension (e.g. 5000 for LC-MS depending on acquisition
-rate), the threshold should be lowered to 80 or less.
 
-##### Search minimum range
+3. _Mobilograms_ contain less scans (_e.g._ ≈400-1000 per [frame](../../terminology/ion-mobility-terminology.md#accumulations-mobility-scans-and-frames), depending on instrument type and acquisition settings) compared to regular EICs (_e.g._ ≈4500 scans for 15 minutes LC run and scan rate of 0.2 seconds). Therefore, a lower [chromatographic threshold](#chromatographic-threshold) (_e.g._ 80%) is recommended to avoid relevant data points in the mobilogram being discarded.
 
-This value determines the search range in mobility dimension. Therefore, this value has to be set
-lower when resolving a TIMS (Bruker data) mobilogram, because the absolute numerical values are
-smaller (e.g., 0.01). When resolving mobilograms from Waters or Agilent data (mobility as drift time
-in ms), the values are higher and must therefore be increased.
+    On the other hand, a single feature in the IM dimension is normally made up of more data points than regular LC peaks, due to the different timescale the IM separation is performed on (see [here](../../terminology/ion-mobility-terminology.md#background) for more details). Therefore, a higher _Min # of data points_ can be set when [resolving the ion mobility dimension](#resolving-the-ion-mobility-dimension) to filter out noisy features.
 
-Other parameters such as minimum intensities or minimum number of data points should be adjusted
-depending on what you want to detect.
+4. Different vendors use different units of mobility. For instance, [TIMS](../../terminology/ion-mobility-terminology.md#trapped-ion-mobility-spectrometry-tims) express ions' mobility as Vs/cm<sup>2</sup>, whereas [time dispersive IM devices](../../terminology/ion-mobility-terminology.md#time-dispersve-ion-mobility-spectrometry-dtims-and-twims) (DTIMS and TWIMS) use the ions' drift time (expressed in milliseconds). TIMS values are numerically smaller (_e.g._ 0.01) than DTIMS or TWIMS (_e.g._ 0.05); therefore, the [minimum search range](#minimum-search-range-rtmobility-absolute) parameter should be adjusted accordingly. 
```

Comment:
0.05 -> 1 for twims/dtims
twims dimension is something in the range of 10-30 ms
dtims is something in the range of 10-60 ms